watertap/unit_models/leaky_pressure_exchanger.py

```diff
@@ -0,0 +1,491 @@
+        **Valid values:** {
+        see property package for documentation.}"""))
+
+    def build(self):
+        super().build()
+
+        # Pressure exchanger supports only liquid phase
+        if self.config.property_package.phase_list != ['Liq']:
+            raise ConfigurationError(
+                "Pressure exchanger model only supports one liquid phase ['Liq'],"
+                "the property package has specified the following phases {}"
+                    .format(self.config.property_package.phase_list))
+
+        self.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+
+        units_meta = self.config.property_package.get_metadata().get_derived_units
+
+        self.efficiency_pressure_exchanger = Var(
+            self.flowsheet().config.time,
+            initialize=0.95,
+            bounds=(1e-6, 1),
+            domain=NonNegativeReals,
+            units=pyunits.dimensionless,
+            doc='Pressure exchanger efficiency')
+
+        self.solute_leakage_fraction = Var(
+            self.flowsheet().config.time,
+            initialize=0.05,
+            bounds=(1e-6, 1),
+            domain=NonNegativeReals,
+            units=pyunits.dimensionless,
+            doc='Pressure exchanger solute leakage')
+
+        self.solvent_leakage_fraction = Var(
+            self.flowsheet().config.time,
+            initialize=0.05,
+            bounds=(1e-6, 1),
+            domain=NonNegativeReals,
+            units=pyunits.dimensionless,
+            doc='Pressure exchanger solvent leakage')
+        # Build control volume for high pressure side
+        self.high_pressure_side = ControlVolume0DBlock(default={
+            "dynamic": False,
+            "has_holdup": False,
+            "property_package": self.config.property_package,
+            "property_package_args": self.config.property_package_args})
+
+        self.high_pressure_side.add_state_blocks(
+            has_phase_equilibrium=False)
+
+        self.high_pressure_side.add_material_balances(
+            balance_type=self.config.material_balance_type,
+            has_mass_transfer=True)
+
+        self.high_pressure_side.add_momentum_balances(
+            balance_type=self.config.momentum_balance_type,
+            has_pressure_change=True)
+
+        self.high_pressure_side.deltaP.setub(0)
+
+        @self.high_pressure_side.Expression(
+            self.flowsheet().config.time,
+            doc='Work transferred to high pressure side fluid (should be negative)')
+        def work(b, t):
+            return b.properties_in[t].flow_vol * b.deltaP[t]
+
+        @self.high_pressure_side.Expression(
+            self.flowsheet().config.time,
+            self.config.property_package.phase_list,
+            self.config.property_package.component_list,
+            doc='Mass transfered on high pressure side')
+        def mass_transfer(b, t, p, j):
+            return b.properties_in[t].flow_mass_phase_comp[p,j]-b.properties_out[t].flow_mass_phase_comp[p,j]
```

Comment:
As above, the control volume is already creating this for you, so you are now replacing the existing Var.

Reply:
Will remove then.

Reply:
will fix